interop/src/main.rs

```diff
@@ -180,9 +185,10 @@ async fn main() {
             .collect();
 
         peers.iter_mut().for_each(|mut x| {
-            match (opt.h3, opt.hq) {
-                (false, true) => x.alpn = Alpn::Hq,
-                (true, false) => x.alpn = Alpn::H3,
+            match (opt.h3, opt.hq, opt.siduck) {
+                (false, true, _) => x.alpn = Alpn::Hq,
+                (true, false, _) => x.alpn = Alpn::H3,
+                (_, _, true) => x.alpn = Alpn::Siduck,
```

Comment:
This looks suspiciously close to the other one. DRY? (I know it's not new, but I'll invoke the boy scout rule.)